### ISSUE TITLE
중복검색 원인 추측 : created: 옵션이 잘못 됐을수 있습니다.

### ISSUE THREAD

**Issue by phg98** [taetaetae/github-repo-searcher#9]
회사라 돌려보지는 못하고 궁예만 합니다.ㅠㅠ
중복결과가 나오는 이유가 혹시 이것이 아닐까 의심해봅니다.

서치 스트링에 created:{search_base_time} 이렇게 되어있는데 이 포맷은 문서에 없습니다.
날짜앞에 >, >=, <, <=가 있거나 아니면 range로 지정해야 합니다. [링크참조](https://docs.github.com/en/free-pro-team@latest/github/searching-for-information-on-github/understanding-the-search-syntax)

그러니까 하기 세개중의 하나가 되어야 할 것 같고
* created:**<**{search_base_time} (또는 <=)
* created:**>**{search_base_time} (또는 >=)
* created:{search_base_time1}**..**{search_base_time2}

저희의 목적에는 세번째 항목인 레인지 기반 검색을 하는 것이 적절할 것 같습니다.

**Comment by taetaetae** [comment]
- https://github.com/search/advanced?q=created%3A2020-10-12&type=Repositories 에서 해보시면 `:` 도 허용 됩니다.
![image](https://user-images.githubusercontent.com/10969354/95744459-48da6280-0cce-11eb-8c3b-d2c87c941516.png)

-그리고 말씀하신 `created:{search_base_time1}..{search_base_time2}` 도 고려 해봤지만 범위가 길어질 수록 양이 많아져 제한값 이상을 조회하기 어렵습니다.(1,000) 

- 같은 날짜에 page 를 달리하고서 호출을 해보면 아래와 같은 결과를 얻을 수 있는데요.
![image](https://user-images.githubusercontent.com/10969354/95744741-ce5e1280-0cce-11eb-946d-87b58caa4ded.png)
![image](https://user-images.githubusercontent.com/10969354/95744888-14b37180-0ccf-11eb-93f2-590563ebcdb4.png)
짐작이지만, 검색 서버의 색인하는 타이밍과 해당 repo 의 데이터 변경되는 타이밍이 맞지 않아 간헐적으로 오류가 있어 보입니다. (검색 색인서버의 오류가 아닐까 싶은...)

**Comment by phg98** [comment]
네, 그냥 날짜만 쓰면 하룿ㅣ가 찾아지는군요. 하루에 엄청난 수의 저장소가 생기네요. 그러고 보니 저희는 저장소 먼저 검색하노 로케이션 검색하는데, 반대로 로케이션으로 사용자검색먼저 하고 그 안에서 저장소 검색하면 더 효율적이지 않을까 하는 생각도 듭니다. 최근 일년동안  생성된 한국인유저는 4k정도니까 저장소보다 훨씬  작지 않을지. 구미 출장왔는데 버스에서 핸드폰 으로 쓰니까 멀미납니다. ㅠㅠ

**Comment by taetaetae** [comment]
사실

**Comment by phg98** [comment]
사용자 먼저 검색하는 것을 별도 이슈로 생성하겠습니다.
